### PR TITLE
Add props to set custom min and max zoom levels

### PR DESCRIPTION
**Pull request by MattyBalaam**
This pull request brings in customisable minimum and maximum scrollwheel zoom values.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -164,7 +164,7 @@ class Cropper extends React.Component {
   }
 
   setNewZoom = zoom => {
-    const newZoom = Math.min(MAX_ZOOM, Math.max(zoom, MIN_ZOOM))
+    const newZoom = Math.min(this.props.minZoom, Math.max(zoom, this.props.maxZoom))
```
Comment (ValentinH) [inline src/index.js]: you switched min and max here ;)

Comment (ValentinH) [reply src/index.js]: Thanks to the e2e tests to detect this. 🙂

**Comment by ValentinH** [comment]
Nice, I was sure that somebody was going to contribute this 😆.

File: src/index.js (hunk 1 above)
Comment (MattyBalaam) [reply src/index.js]: Ah, sorry, will recommit

**Comment by ValentinH** [comment]
Thanks!

**Pull request merged**